frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -135,7 +137,7 @@ const rerunPipeline: KebabAction = (
 ) => {
   const { onComplete } = customData;
 
-  const sharedProps = { label: 'Start Last Run', accessReview: {} };
+  const sharedProps = { label: 'Rerun', accessReview: {} };
```

Comment:
According to @serenamarie125 this should be `Start Last Run` for Pipeline. Please adjust accordingly. See the JIRA ticket for the response.

Reply:
yes, this was earlier adjusted to be `Start Last Run` as per  https://jira.coreos.com/browse/ODC-1245

Reply:
Change it back to ```Start Last Run``` for the pipeline.